typescript/create-onchain-agent/templates/next/app/api/agent/evm/viem/route.ts

```diff
@@ -16,7 +16,7 @@ import { ChatOpenAI } from "@langchain/openai";
 import { MemorySaver } from "@langchain/langgraph";
 import { createReactAgent } from "@langchain/langgraph/prebuilt";
 import { AgentRequest, AgentResponse } from "@/app/types/api";
-import { createWalletClient, http } from "viem";
+import { Chain, createWalletClient, http } from "viem";
```

Comment:
This looks unused, does it end up getting used down the line?

Reply:
Good catch! It was used when I have `evm/viem` and `custom-evm/viem` as one case earlier in this PR. It became unused when I refactored them into two files